### PR TITLE
ENG-5289 #2025 [YSQL][docs] DO statement

### PR DESCRIPTION
**Pull request by d-uspenskiy**
Add description for `DO` command into `latest/api/ysql/commands/` YB docs section.
Description and example are taken from Postgres's official doc 

Test plan:
Manual
- start docs web browser
- check `do` command in command list at `latest/api/ysql/commands/` URL
- check link for `do` is clickable and leads to working page with detailed description

### CODE REVIEW

File: docs/.gitignore [hunk 1]
```diff
@@ -4,6 +4,7 @@
 public/*
 npm-debug.log
 node_modules
+package-lock.json
```
[the remark this answers is not in the document]
Comment (d-uspenskiy) [reply docs/.gitignore]: As far as I understand `package-lock.json` relates to `npm` package local install process. It should not be tracked

[the remark this answers is not in the document]
Comment (d-uspenskiy) [reply docs/.gitignore]: Done. package-lock.json is now removed

File: docs/content/latest/api/ysql/_index.md [hunk 2]
```diff
@@ -107,6 +107,7 @@ Note that Serializable isolation level for multi-key transactions was added in [
 | Statement | Description |
 |-----------|-------------|
 | [`COPY`](commands/cmd_copy) | Copy data between tables and files |
+| [`DO`](commands/cmd_doc) | Execute an anonymous code block |
```
Comment (m-iancu) [inline docs/content/latest/api/ysql/_index.md]: why is is `cmd_doc` rather than `cmd_do`?

File: docs/content/latest/api/ysql/commands/_index.md [hunk 3]
```diff
@@ -35,6 +35,7 @@ The following table lists the commands in YSQL.
 | [`CREATE VIEW`](ddl_create_view) | Create a new view |
 | [`DEALLOCATE`](perf_deallocate) | Deallocate a prepared statement |
 | [`DELETE`](dml_delete) | Delete rows from a table |
+| [`DO`](cmd_doc) | Execute an anonymous code block |
```
Comment (m-iancu) [inline docs/content/latest/api/ysql/commands/_index.md]: same here.

File: docs/content/latest/api/ysql/commands/cmd_do.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+title: DO
+linkTitle: DO
+summary: DO
+description: DO
+menu:
+  latest:
+    identifier: api-ysql-commands-do
+    parent: api-ysql-commands
+aliases:
+  - /latest/api/ysql/commands/cmd_do
```
Comment (m-iancu) [inline docs/content/latest/api/ysql/commands/cmd_do.md]: nit: I don't think you need the alias here. That's only for backwards compatibility with old/changed links.
But please double-check that -- if unsure it's probably safe to leave it.

File: docs/.gitignore (hunk 1 above)
[the remark this answers is not in the document]
Comment (m-iancu) [reply docs/.gitignore]: Talked to @yb-andrew -- lets not remove/gitignore `package-lock.json`. Instead just update it with your changes as caused by adding the `hugo` deps. Will let @yb-andrew  review that part once it's updated.

[the remark this answers is not in the document]
Comment (jj-kim) [reply docs/.gitignore]: Please don't create a new pull request for this (like #2024 to #2214).  You can `git revert 49dd0c7` and edit `.gitignore` as two separate commits on top of the two existing ones, for example.

**Pull request closed without merging**